src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+
+    public boolean validate(String password,
+                            String repeatPassword) throws PasswordValidationException {
+
+        if (password != null && repeatPassword != null
+                && repeatPassword.length() >= 10 && password.length() >= 10
+                && password.equals(repeatPassword)) {
+            return true;
+        } else {
```

Comment:
do we need `else` block after `return` keyword?

Reply:
We don't need it.